scripts/20230529_rm_payment_enabled/script.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable */
+
+// BEFORE
+// Count total number of forms with payment_fields that have enabled value. This should be > 0.
+db.getCollection('forms').countDocuments({ "payments_field.enabled": { $exists: true } })
+
+// UPDATE - remove payments_field.enabled
+db.getCollection('forms').updateMany(
+  {},
```

Comment:
Reduce the number of documents to update (even if the updates are inconsequential):
```suggestion
  { "payments_field.enabled": { $exists: true } },
```

Also just to confirm: running this script is just cleanup. It's inconsequential to the code, since we are removing awareness of the existence of the property, correct?

Reply:
Nope, validation will fail if we don't do this because when we try to save the payment field through the input panel, it takes the whole `payment_field`, updates the fields based on the form values, and tries to chuck it all back to BE to deal with it. So the BE will throw an err because of the unexpected `enabled` field.

Reply:
Ah dang it! Thanks for the clarification @LinHuiqing 🌮 

So it makes the timing of the release a bit stressful. As soon as the new backend code is deployed, we need to run this script, or we'll be seeing errors when admins edit their forms. Since payment is in beta, I think we know the volume will be low enough to not have problems, but if we have to do this again when payment is more broadly  adopted, we will need to have a backward compatibility plan.

How interesting 😑 , a document db gives us limitless flexibility to have extra fields that we don't care about and move fast, but our strict typing in code is in the way of our velocity.

Reply:
Yea, I think its fine because payment features are still behind payment feature flag for now! So basically, we need to run this script to clear it then we can enabled the feature flag.